src/terms/hubbard.jl

```diff
@@ -107,51 +107,75 @@ Hubbard energy, following the Dudarev et al. (1998) rotationally invariant forma
 ```
 """
 struct Hubbard{T}
-    manifold::OrbitalManifold
-    U::T
-    function Hubbard(manifold::OrbitalManifold, U)
-        U = austrip(U)
-        new{typeof(U)}(manifold, U)
+    manifolds::Vector{OrbitalManifold}
+    U::Vector{T}
+    function Hubbard(manifolds::Vector{OrbitalManifold}, U)
+        U = austrip.(U)
+        new{typeof(U[1])}(manifolds, U)
+    end
+    function Hubbard(manifold_to_U::Vararg{<:Pair})
+        manifolds = map(p -> p[1], manifold_to_U)
+        Us = map(p -> p[2], manifold_to_U)
+        Hubbard([manifold for manifold in manifolds], [U for U in Us])
+    end
```

Comment:
Make this a stand-alone function, also
```suggestion
Hubbard(manifold_to_U::Pair...) = Hubbard(first.(manifold_to_U), last.(manifold_to_U))
```
should do the same thing, but is a bit nicer.

Reply:
I'm not sure if it is my problem but this doesn't work on my computer, it doesn't accept the usual (manifold => U) input.